### ISSUE TITLE
Stop creating issues - this project is dead!

### ISSUE THREAD

**Issue by ololoe** [dlitz/pycrypto#173]
Move to a fork like `pycryptodome`

**Comment by pkoning2** [comment]
Is there any reason to believe that the original comment is valid?

**Comment by mouse07410** [comment]
"Valid" in what sense? `pycryptodome` is a fork that appears to be maintained, and contains algorithms that the original PyCrypto has been missing. I've started using it, and so far so good.

There hasn't been a new release of PyCrypto for several years. 2.7 has been in alpha state for approximately that long. So while I cannot tell whether the original project is **dead**, it clearly **isn't actively maintained**.

**Comment by johnthagen** [comment]
Took me a while to figure this out -- maybe the README should be updated to clearly state this at the top?

**Comment by pkoning2** [comment]
Yes, that would be good.  For one thing, that would alert other projects that depend on pycrypto (like pysnmp) that they need to switch, or at the very least that they need to be able to use pycryptome as an alternative dependency.

**Comment by ololoe** [comment]
Holy fucking shit, it even has an exploitable buffer overflow (although no idea how to do it remotely) https://github.com/dlitz/pycrypto/issues/176

**Comment by WGH-** [comment]
@ololoe https://pony7.fr/ctf:public:32c3:cryptmsg

**Comment by adombeck** [comment]
@dlitz Could you please update the README to reflect that this project has been unmaintained for almost 3 years?

**Comment by Jamim** [comment]
bump
